Lab4/src/ca/mcgill/ecse211/lab4/Lab4.java

```diff
@@ -0,0 +1,141 @@
+package ca.mcgill.ecse211.lab4;
+
+import lejos.hardware.Button;
+import lejos.hardware.ev3.LocalEV3;
+import lejos.hardware.lcd.TextLCD;
+import lejos.hardware.motor.EV3LargeRegulatedMotor;
+import lejos.hardware.port.Port;
+import lejos.hardware.sensor.EV3UltrasonicSensor;
+import lejos.hardware.sensor.SensorModes;
+import lejos.robotics.SampleProvider;
+
+/**
+ * This is the main class for our program. It starts the program
+ * and it allows the user to select different options on how they want the run 
+ * to work (either run the demo or our test programs)
+ * It has been slightly modified from the version given in the lab 2 documents to fit this lab
+ *
+ * {@value #leftMotor}  value from the left motor
+ * {@value #rightMotor} value from the right motor
+ * {@value #WHEEL_RAD}  constant, value of the wheel's radius
+ * {@value #TRACK}      constant, value of track
+ * {@value #SPEED}      constant, moving speed of robot
+ * {@value #usPort}     value from usPort
+ * 
+ * @author Maxime Bourassa
+ * @author Violet Wei
+ *
+ */
+public class Lab4 {
+
+  // Motor Objects, and Robot related constant parameters
+  //These all have getters to be accessed from other classes

```

Comment:
This comment is no longer true. Since they're public, you can access them directly, and that's what you are doing, and that what you should be doing.